plugins/sfPluginAdminPlugin/modules/sfPluginAdminPlugin/actions/pluginsAction.class.php

```diff
@@ -112,6 +114,13 @@ public function execute($request)
         $cacheClear = new sfCacheClearTask(sfContext::getInstance()->getEventDispatcher(), new sfFormatter());
         $cacheClear->run();
 
+        // Notify use if SWORD setting has been changed
+        if ($swordEnabled != in_array('qtSwordPlugin', $settings))
+        {
+          $message = $this->context->i18n->__('SWORD plugin setting changed: the AtoM worker must be restarted for this change to take effect.');
+          $this->getUser()->setFlash('error', $message);
```

Comment:
Maybe `info` (not sure if that's the actual term used in this flash system) or `warning` are better than `error`. I'd go with `info` in this case ;)

Reply:
Thanks Radda! Makes sense... I'll change that.